lib/ws-kernel-picker.js

```diff
@@ -84,7 +97,7 @@ export default class WSKernelPicker {
     });
   }
 
-  onGateway(gatewayInfo) {
+  onGateway(gatewayInfo: any) {
```

Comment:
I wasn't familiar enough with the intended objects here and below to type them. Took a couple shots, but decided not to get too caught up. 

Open to input or we could just keep these as a todo.